lib/Frontend/FrontendOptions.cpp

```diff
@@ -227,6 +263,7 @@ bool FrontendOptions::canActionEmitObjCHeader(ActionType action) {
   case ActionType::REPL:
     return false;
   case ActionType::Parse:
+  case ActionType::ResolveImports:
```

Comment:
Actually, I think Parse and ResolveImports belong in the "false" section for this one. It definitely needs a type-checked AST.

Reply:
I'll submit another patch for that one.